src/edgepi/digital_output/digital_output_constants.py

```diff
@@ -15,3 +15,11 @@ class DoutPins(Enum):
     DOUT6 = 'DOUT6'
     DOUT7 = 'DOUT7'
     DOUT8 = 'DOUT8'
+
+@unique
+class DoutTriState(Enum):
+    """Digital Output Tri-state Enum"""
+
+    Z = -1
```

Comment:
please change to `Hi-Z`

Reply:
c64b58fe15dd94139db7e678f51a964dd20c790c